src/XacroParser.js

```diff
@@ -211,17 +211,18 @@ export class XacroParser {
             // strip the asterisks
             param = param.replace(/^\*{1,2}/g, '');
 
-            // Check if a default value is provided
-            if (/:=/.test(param)) {
-                const [name, def] = param.split(':=');
+            // Check if a default value is provided (= or := syntax)
+            if (param.includes('=')) {
```

Comment:
I think I'd prefer to change this to `/:?=/.test(param)` so it's clear it's still testing for both `:=` and `=`. The same regex can be used to split the param, as well, which also would mean the replace operation used to remove the dangling ":" can be removed.